.travis.yml

```diff
@@ -19,8 +19,11 @@ services:
 before_install:
   - mkdir -p $HOME/.hypothesis
   # If necessary, set up an appropriate version of pypy.
-  - if [ ! -z "$PYPY_VERSION" ]; then source setup-pypy-travis.sh; fi
-  - if [ ! -z "$PYPY_VERSION" ]; then python --version 2>&1 | fgrep "PyPy $PYPY_VERSION"; fi
+  - >
+     if [[ -n "$PYPY_VERSION" ]]; then
```

Comment:
Travis's documentation is a bit sketchy in places. The best I can find is this: https://docs.travis-ci.com/user/customizing-the-build/#Note-on-%24%3F *"Each command in script is processed by a special bash function"*